proto/storage/relations.proto

```diff
@@ -56,12 +56,6 @@ message NodeComponentEdge {
   string node_component_id = 3 [(gogoproto.moretags) = 'sql:"pk,fk(ImageComponent:id),no-fk-constraint"'];
 }
 
-message NodeCVEEdge {
```

Comment:
do we not need to deprecate this all first or have we not been using it for a while?